vector/src/main/res/layout/view_stub_room_profile_header.xml

```diff
@@ -54,27 +54,9 @@
         android:textAppearance="@style/Vector.Toolbar.Title"
         android:textSize="14sp"
         android:textStyle="bold"
-        app:layout_constraintBottom_toTopOf="@+id/roomProfileTopicView"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/roomProfileNameView"
         tools:text="@sample/matrix.json/data/roomAlias" />
 
-    <TextView
-        android:id="@+id/roomProfileTopicView"
-        android:layout_width="0dp"
-        android:layout_height="wrap_content"
-        android:layout_marginStart="40dp"
-        android:layout_marginEnd="40dp"
-        android:autoLink="web"
```

Comment:
We have lost this, please restore in the new layout

Reply:
Done.

Reply:
You added back the attributes to the ImageView and not the TextView... I will fix it.